FormlyMapper/FormlyMapper.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace Linio\DynamicFormBundle\FormlyMapper;
+
+use Linio\DynamicFormBundle\Exception\NonExistentFormException;
+use Linio\DynamicFormBundle\Exception\FormlyMapperException;
+use Linio\DynamicFormBundle\Form\FormFactory;
+use Linio\DynamicFormBundle\FormlyMapper\FormlyField\FormlyFieldFactory;
+use Symfony\Component\Security\Csrf\CsrfTokenManagerInterface;
+
+class FormlyMapper
+{
+    /**
+     * @var CsrfTokenManagerInterface
+     */
+    protected $csrfTokenManager;
+
+    /**
+     * @var FormFactory
+     */
+    protected $formFactory;
+
+    /**
+     * @var FormlyFieldFactory
+     */
+    protected $formlyFieldFactory;
+
+    /**
+     * @param CsrfTokenManagerInterface $csrfTokenManager
+     */
+    public function setCsrfTokenManager(CsrfTokenManagerInterface $csrfTokenManager)
+    {
+        $this->csrfTokenManager = $csrfTokenManager;
+    }
+
+    /**
+     * @param FormFactory $formFactory
+     */
+    public function setFormFactory(FormFactory $formFactory)
+    {
+        $this->formFactory = $formFactory;
+    }
+
+    /**
+     * @param FormlyFieldFactory $formlyFieldFactory
+     */
+    public function setFormlyFieldFactory(FormlyFieldFactory $formlyFieldFactory)
+    {
+        $this->formlyFieldFactory = $formlyFieldFactory;
+    }
+
+    /**
+     * @param string $formName
+     *
+     * @return array
+     * @throws FormlyMapperException
+     */
+    public function map($formName = null)
+    {
+        $formlyConfiguration = [];
+
+        try {
```

Comment:
Since you are only catching `NonExistentFormException`, why not just have the `$configuration = (array) ...;` line in the try/catch block? The rest of that shouldn't be throwing a `NonExistentFormException`. If they are, it's the wrong Exception to be throwing.